src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php

```diff
@@ -61,13 +61,18 @@ public function getImageAction(Request $request)
             }
 
             $url = $request->getPathInfo();
+            // there are some projects which do not call this action with ?v=1-0 because of they didn't wanted query strings in the image urls ( unnecessary SEO mystic reasons )
```

Comment:
```suggestion
            // Some projects do not call this action with ?v=1-0 because they don't want query strings in the image urls ( unnecessary SEO mystic reasons )
```